README.md

```diff
@@ -121,51 +119,44 @@ To see a diagram of the dependencies of your projects:
 yarn nx dep-graph
 ```
 
-### Making dev secrets available locally
+### AWS Secrets
 
-Environment variables that should not be tracked but needed locally should be added to the `.env.secret` file.  
-_(**NOTE:** Each variable must be prefixed with `export ` for direnv to pick them up.)_
+A dedicated documentation about fetching shared development secrets or creating new secrets, using AWS secrets is available [here](handbook/repository/aws-secrets.md).
 
-Additionally, if that same variable is also stored in AWS Parameter Store, the secret can be labeled with the `dev` label from `History` -> `Attach labels`.
+### Running proxy against development service
 
-All secrets labeled with the `dev` label can be fetched using `yarn get-secrets`.
+You are able to run a couple of proxy against the development services: `db`, `es`, `redis` and `xroad`.
 
-### Fetch development secrets for your project
+To do so, you can run for example:
 
 ```bash
-yarn get-secrets <project> [options]
+./scripts/run-db-proxy.sh
 ```
 
-**Example**:
+It will try to get your AWS credentials through the bash EXPORTS or either defined in the `~/.aws/credentials` file. If you chose the AWS SSO method to use the AWS scripts that won't work for you. You will need to visit your AWS account and export or edit the `~/.aws/credentials` file. More instructions available [here](handbook/repository/aws-secrets.md#using-aws-session).
```

Comment:
```suggestion
It will try to get your AWS credentials from your environment variables and from your `~/.aws/credentials` file. We're currently not able to get AWS credentials from an AWS profile configured using AWS SSO. You will need to visit your AWS account and export the credentials manually or add them to your `~/.aws/credentials` file. You can find more instructions [here](handbook/repository/aws-secrets.md#using-aws-session).
```